mpmath/matrices/linalg.py

```diff
@@ -134,6 +134,9 @@ def LU_decomp(ctx, A, overwrite=False, use_cache=True):
                 if current > biggest: # TODO: what if equal?
                     biggest = current
                     p[j] = k
+            # without pivot LU fails
+            if p[j] is None:
```

Comment:
Is this what happens for the test case `A11`?

Reply:
yes